From c43b89123d59b0883ca59668ca7f38f465dff6b8 Mon Sep 17 00:00:00 2001
From: Alan Tull <alan.tull@freescale.com>
Date: Wed, 23 Feb 2011 11:07:16 -0600
Subject: [PATCH 210/449] ENGR00139671 change autorun scripts to specify bash

Scripts were specifying #!/bin/sh, which defaults to bash on our
ltib release root filesystem and to dash on Ubuntu.  Change to
specify #!/bin/bash so that the scripts will work for any Linux
that has bash.

Signed-off-by: Alan Tull <alan.tull@freescale.com>
---
 autorun.sh                                       | 2 +-
 readme.txt                                       | 2 +-
 test-utils.sh                                    | 2 +-
 test/max8660_pmic_test/autorun-mcu_pmic.sh       | 2 +-
 test/mxc_asrc_test/autorun-asrc.sh               | 2 +-
 test/mxc_dryice_test/dryice.sh                   | 4 ++--
 test/mxc_fb_test/autorun-fb.sh                   | 2 +-
 test/mxc_fb_test/mxc_tve_test.sh                 | 2 +-
 test/mxc_firi_test/autorun-firi.sh               | 2 +-
 test/mxc_firi_test/mxc_irlan_test.sh             | 2 +-
 test/mxc_hdd_test/autorun-ata.sh                 | 2 +-
 test/mxc_i2c_slave_test/autorun-mxc_i2c_slave.sh | 2 +-
 test/mxc_keyb_test/autorun-keypad.sh             | 2 +-
 test/mxc_keyb_test/mxc_keyb_test.sh              | 2 +-
 test/mxc_mbx_test/autorun-mbx.sh                 | 2 +-
 test/mxc_mmc_test/autorun-mmc.sh                 | 2 +-
 test/mxc_mpeg4_enc_test/autorun-hantro.sh        | 2 +-
 test/mxc_oprofile_test/autorun-oprofile.sh       | 2 +-
 test/mxc_owire_test/autorun-owire.sh             | 2 +-
 test/mxc_pf_test/autorun-pf.sh                   | 2 +-
 test/mxc_pmic_test/autorun-mc34704.sh            | 2 +-
 test/mxc_pmic_test/autorun-pmic.sh               | 2 +-
 test/mxc_rng_test/autorun-rng.sh                 | 2 +-
 test/mxc_rtc/autorun-rtc.sh                      | 2 +-
 test/mxc_sahara_test/autorun-sahara.sh           | 2 +-
 test/mxc_scc/autorun-scc.sh                      | 2 +-
 test/mxc_scc/encrypt_decrypt_tests.sh            | 2 +-
 test/mxc_scc/fail_mode_tests.sh                  | 2 +-
 test/mxc_scc/key_slot_tests.sh                   | 2 +-
 test/mxc_scc/scc_setup.sh                        | 2 +-
 test/mxc_scc/test_subs.sh                        | 2 +-
 test/mxc_scc2/autorun-scc2.sh                    | 2 +-
 test/mxc_scc2/fail_mode_tests.sh                 | 2 +-
 test/mxc_scc2/partition_tests.sh                 | 2 +-
 test/mxc_scc2/scc2_setup.sh                      | 2 +-
 test/mxc_scc2/test_subs.sh                       | 2 +-
 test/mxc_sound_test/autorun-sound.sh             | 2 +-
 test/mxc_uart_test/autorun-mxc_uart.sh           | 2 +-
 test/mxc_usb_test/autorun-usb-gadget.sh          | 2 +-
 test/mxc_usb_test/autorun-usb-host.sh            | 2 +-
 test/mxc_v4l2_test/autorun-v4l2.sh               | 2 +-
 test/mxc_v4l2_test/emma_v4l2_loopback.sh         | 2 +-
 test/mxc_v4l2_test/mxc_v4l2_loopback.sh          | 2 +-
 test/mxc_v4l2_test/mxc_v4l2_out_testall.sh       | 2 +-
 test/mxc_vpu_test/autorun-vpu.sh                 | 2 +-
 test/wdog/autorun-wdog.sh                        | 2 +-
 46 files changed, 47 insertions(+), 47 deletions(-)

diff --git a/autorun.sh b/autorun.sh
index e511f96..a76e01a 100755
--- a/autorun.sh
+++ b/autorun.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 # If this script is run with no parameters, it will run all the test ID's in
 # autorun-suite.txt.  The autorun-suite.txt file also describes how each test
diff --git a/readme.txt b/readme.txt
index e71aff2..bd8a4ad 100644
--- a/readme.txt
+++ b/readme.txt
@@ -78,7 +78,7 @@ tests are having difficulty.  These scripts can also make it easier for people w
 are not familiar with the unit tests to run them.  Currently the simplest autorun
 script is misc/test/wdog/autorun-wdog.sh, which is quoted below.
 
-      #!/bin/sh
+      #!/bin/bash
 
       source /unit_tests/test-utils.sh
 
diff --git a/test-utils.sh b/test-utils.sh
index 328e46f..d8ce686 100755
--- a/test-utils.sh
+++ b/test-utils.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #
 # If one test case fails, this script keeps running the rest of the
diff --git a/test/max8660_pmic_test/autorun-mcu_pmic.sh b/test/max8660_pmic_test/autorun-mcu_pmic.sh
index a5873c7..fb6cda0 100644
--- a/test/max8660_pmic_test/autorun-mcu_pmic.sh
+++ b/test/max8660_pmic_test/autorun-mcu_pmic.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_asrc_test/autorun-asrc.sh b/test/mxc_asrc_test/autorun-asrc.sh
index 553c987..aeb4e7f 100755
--- a/test/mxc_asrc_test/autorun-asrc.sh
+++ b/test/mxc_asrc_test/autorun-asrc.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_dryice_test/dryice.sh b/test/mxc_dryice_test/dryice.sh
index 2679464..2842727 100644
--- a/test/mxc_dryice_test/dryice.sh
+++ b/test/mxc_dryice_test/dryice.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 PATH=$PATH:/unit_tests/modules:$PWD
 source /unit_tests/test-utils.sh
 
@@ -31,7 +31,7 @@ run()
 	fi
 	n_total=$(($n_total+1))
 }
-	
+
 run test=setp nr=0
 run test=setp nr=1
 run test=setp nr=2
diff --git a/test/mxc_fb_test/autorun-fb.sh b/test/mxc_fb_test/autorun-fb.sh
index 9b8da67..a502edd 100755
--- a/test/mxc_fb_test/autorun-fb.sh
+++ b/test/mxc_fb_test/autorun-fb.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_fb_test/mxc_tve_test.sh b/test/mxc_fb_test/mxc_tve_test.sh
index 172d150..21b65b0 100755
--- a/test/mxc_fb_test/mxc_tve_test.sh
+++ b/test/mxc_fb_test/mxc_tve_test.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 echo Setting TV to NTSC mode
 echo U:720x480i-60 > /sys/class/graphics/fb1/mode
diff --git a/test/mxc_firi_test/autorun-firi.sh b/test/mxc_firi_test/autorun-firi.sh
index ff75a73..fc093d9 100755
--- a/test/mxc_firi_test/autorun-firi.sh
+++ b/test/mxc_firi_test/autorun-firi.sh
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_firi_test/mxc_irlan_test.sh b/test/mxc_firi_test/mxc_irlan_test.sh
index 7403190..9afdfe0 100755
--- a/test/mxc_firi_test/mxc_irlan_test.sh
+++ b/test/mxc_firi_test/mxc_irlan_test.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 kernel_ver=2.6.22
 
diff --git a/test/mxc_hdd_test/autorun-ata.sh b/test/mxc_hdd_test/autorun-ata.sh
index de315d8..055bc17 100755
--- a/test/mxc_hdd_test/autorun-ata.sh
+++ b/test/mxc_hdd_test/autorun-ata.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #   On imx31ads board, turn off buzzer (set SW2-2 to OFF).
 
diff --git a/test/mxc_i2c_slave_test/autorun-mxc_i2c_slave.sh b/test/mxc_i2c_slave_test/autorun-mxc_i2c_slave.sh
index 44d574d..ec82b86 100755
--- a/test/mxc_i2c_slave_test/autorun-mxc_i2c_slave.sh
+++ b/test/mxc_i2c_slave_test/autorun-mxc_i2c_slave.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_keyb_test/autorun-keypad.sh b/test/mxc_keyb_test/autorun-keypad.sh
index bd86500..8e98eb9 100755
--- a/test/mxc_keyb_test/autorun-keypad.sh
+++ b/test/mxc_keyb_test/autorun-keypad.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_keyb_test/mxc_keyb_test.sh b/test/mxc_keyb_test/mxc_keyb_test.sh
index 82d1f1c..81f259a 100755
--- a/test/mxc_keyb_test/mxc_keyb_test.sh
+++ b/test/mxc_keyb_test/mxc_keyb_test.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #keypad test is evtest
 
diff --git a/test/mxc_mbx_test/autorun-mbx.sh b/test/mxc_mbx_test/autorun-mbx.sh
index 01ab604..02216b7 100755
--- a/test/mxc_mbx_test/autorun-mbx.sh
+++ b/test/mxc_mbx_test/autorun-mbx.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_mmc_test/autorun-mmc.sh b/test/mxc_mmc_test/autorun-mmc.sh
index 67ad49d..4b4830b 100755
--- a/test/mxc_mmc_test/autorun-mmc.sh
+++ b/test/mxc_mmc_test/autorun-mmc.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_mpeg4_enc_test/autorun-hantro.sh b/test/mxc_mpeg4_enc_test/autorun-hantro.sh
index 0a3e0d3..83a26b1 100755
--- a/test/mxc_mpeg4_enc_test/autorun-hantro.sh
+++ b/test/mxc_mpeg4_enc_test/autorun-hantro.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_oprofile_test/autorun-oprofile.sh b/test/mxc_oprofile_test/autorun-oprofile.sh
index 3e4a2b7..fbf3bac 100755
--- a/test/mxc_oprofile_test/autorun-oprofile.sh
+++ b/test/mxc_oprofile_test/autorun-oprofile.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_owire_test/autorun-owire.sh b/test/mxc_owire_test/autorun-owire.sh
index 92b33e4..010817b 100755
--- a/test/mxc_owire_test/autorun-owire.sh
+++ b/test/mxc_owire_test/autorun-owire.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_pf_test/autorun-pf.sh b/test/mxc_pf_test/autorun-pf.sh
index d7070e5..cf3a0b5 100755
--- a/test/mxc_pf_test/autorun-pf.sh
+++ b/test/mxc_pf_test/autorun-pf.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_pmic_test/autorun-mc34704.sh b/test/mxc_pmic_test/autorun-mc34704.sh
index 74a6487..1f836dc 100644
--- a/test/mxc_pmic_test/autorun-mc34704.sh
+++ b/test/mxc_pmic_test/autorun-mc34704.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_pmic_test/autorun-pmic.sh b/test/mxc_pmic_test/autorun-pmic.sh
index 52eb262..2b7d8c4 100755
--- a/test/mxc_pmic_test/autorun-pmic.sh
+++ b/test/mxc_pmic_test/autorun-pmic.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_rng_test/autorun-rng.sh b/test/mxc_rng_test/autorun-rng.sh
index 7f74a12..c6b8371 100755
--- a/test/mxc_rng_test/autorun-rng.sh
+++ b/test/mxc_rng_test/autorun-rng.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index ce327fe..f287a61 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_sahara_test/autorun-sahara.sh b/test/mxc_sahara_test/autorun-sahara.sh
index 483605b..8e9d15a 100755
--- a/test/mxc_sahara_test/autorun-sahara.sh
+++ b/test/mxc_sahara_test/autorun-sahara.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_scc/autorun-scc.sh b/test/mxc_scc/autorun-scc.sh
index 726893e..2a89851 100755
--- a/test/mxc_scc/autorun-scc.sh
+++ b/test/mxc_scc/autorun-scc.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 PATH=$PATH:/unit_tests/modules:$PWD
 source /unit_tests/test-utils.sh
diff --git a/test/mxc_scc/encrypt_decrypt_tests.sh b/test/mxc_scc/encrypt_decrypt_tests.sh
index 330183b..8fee645 100755
--- a/test/mxc_scc/encrypt_decrypt_tests.sh
+++ b/test/mxc_scc/encrypt_decrypt_tests.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #set -x
 
diff --git a/test/mxc_scc/fail_mode_tests.sh b/test/mxc_scc/fail_mode_tests.sh
index 3167bd1..e980c11 100755
--- a/test/mxc_scc/fail_mode_tests.sh
+++ b/test/mxc_scc/fail_mode_tests.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 
 ################################################################
diff --git a/test/mxc_scc/key_slot_tests.sh b/test/mxc_scc/key_slot_tests.sh
index 98c8019..8033bfa 100755
--- a/test/mxc_scc/key_slot_tests.sh
+++ b/test/mxc_scc/key_slot_tests.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #set -x
 
diff --git a/test/mxc_scc/scc_setup.sh b/test/mxc_scc/scc_setup.sh
index 69017e2..f585b92 100755
--- a/test/mxc_scc/scc_setup.sh
+++ b/test/mxc_scc/scc_setup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 ## This command should be checked for error status.  It has numerous side
 ## effects relating to exportation of symbols
diff --git a/test/mxc_scc/test_subs.sh b/test/mxc_scc/test_subs.sh
index d81a9a8..a547ae3 100755
--- a/test/mxc_scc/test_subs.sh
+++ b/test/mxc_scc/test_subs.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 ################################################################
 #
diff --git a/test/mxc_scc2/autorun-scc2.sh b/test/mxc_scc2/autorun-scc2.sh
index 20dff3e..9ce28aa 100644
--- a/test/mxc_scc2/autorun-scc2.sh
+++ b/test/mxc_scc2/autorun-scc2.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 PATH=$PATH:/unit_tests/modules:$PWD
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_scc2/fail_mode_tests.sh b/test/mxc_scc2/fail_mode_tests.sh
index 793595d..7d27ff7 100644
--- a/test/mxc_scc2/fail_mode_tests.sh
+++ b/test/mxc_scc2/fail_mode_tests.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # Initialize the test accounting environment
 . test_subs.sh
 
diff --git a/test/mxc_scc2/partition_tests.sh b/test/mxc_scc2/partition_tests.sh
index 6195a05..3c8fb40 100644
--- a/test/mxc_scc2/partition_tests.sh
+++ b/test/mxc_scc2/partition_tests.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 #set -x
 
diff --git a/test/mxc_scc2/scc2_setup.sh b/test/mxc_scc2/scc2_setup.sh
index 1a379d8..a691525 100644
--- a/test/mxc_scc2/scc2_setup.sh
+++ b/test/mxc_scc2/scc2_setup.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 ## This command should be checked for error status.  It has numerous side
 ## effects relating to exportation of symbols
 ##
diff --git a/test/mxc_scc2/test_subs.sh b/test/mxc_scc2/test_subs.sh
index 48e7566..659be18 100644
--- a/test/mxc_scc2/test_subs.sh
+++ b/test/mxc_scc2/test_subs.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 ################################################################
 ## @file test_subs.sh
diff --git a/test/mxc_sound_test/autorun-sound.sh b/test/mxc_sound_test/autorun-sound.sh
index c4eabb9..8ff9178 100755
--- a/test/mxc_sound_test/autorun-sound.sh
+++ b/test/mxc_sound_test/autorun-sound.sh
@@ -1,5 +1,5 @@
 
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_uart_test/autorun-mxc_uart.sh b/test/mxc_uart_test/autorun-mxc_uart.sh
index 41acfb1..3e1bfd0 100755
--- a/test/mxc_uart_test/autorun-mxc_uart.sh
+++ b/test/mxc_uart_test/autorun-mxc_uart.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_usb_test/autorun-usb-gadget.sh b/test/mxc_usb_test/autorun-usb-gadget.sh
index 332f681..04d3bae 100755
--- a/test/mxc_usb_test/autorun-usb-gadget.sh
+++ b/test/mxc_usb_test/autorun-usb-gadget.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_usb_test/autorun-usb-host.sh b/test/mxc_usb_test/autorun-usb-host.sh
index 9c3f54f..d06d90a 100755
--- a/test/mxc_usb_test/autorun-usb-host.sh
+++ b/test/mxc_usb_test/autorun-usb-host.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_v4l2_test/autorun-v4l2.sh b/test/mxc_v4l2_test/autorun-v4l2.sh
index 3437b1f..eb7822e 100755
--- a/test/mxc_v4l2_test/autorun-v4l2.sh
+++ b/test/mxc_v4l2_test/autorun-v4l2.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/mxc_v4l2_test/emma_v4l2_loopback.sh b/test/mxc_v4l2_test/emma_v4l2_loopback.sh
index 0cd44e3..6e24443 100755
--- a/test/mxc_v4l2_test/emma_v4l2_loopback.sh
+++ b/test/mxc_v4l2_test/emma_v4l2_loopback.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Script to loopback MPEG4 encoded capture from camera to screen
 #
diff --git a/test/mxc_v4l2_test/mxc_v4l2_loopback.sh b/test/mxc_v4l2_test/mxc_v4l2_loopback.sh
index 36a902d..e88a5c6 100755
--- a/test/mxc_v4l2_test/mxc_v4l2_loopback.sh
+++ b/test/mxc_v4l2_test/mxc_v4l2_loopback.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 mkfifo /tmp/cap_pipe
 
diff --git a/test/mxc_v4l2_test/mxc_v4l2_out_testall.sh b/test/mxc_v4l2_test/mxc_v4l2_out_testall.sh
index 100ba1b..bb68248 100755
--- a/test/mxc_v4l2_test/mxc_v4l2_out_testall.sh
+++ b/test/mxc_v4l2_test/mxc_v4l2_out_testall.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 ./mxc_v4l2_output.out -iw 352 -ih 240 -ow 176 -oh 120 -d 0 fb-352x240.yuv
 ./mxc_v4l2_output.out -iw 352 -ih 240 -ow 176 -oh 120 -d 0 -r 1 fb-352x240.yuv
diff --git a/test/mxc_vpu_test/autorun-vpu.sh b/test/mxc_vpu_test/autorun-vpu.sh
index 28be057..684fd11 100755
--- a/test/mxc_vpu_test/autorun-vpu.sh
+++ b/test/mxc_vpu_test/autorun-vpu.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
diff --git a/test/wdog/autorun-wdog.sh b/test/wdog/autorun-wdog.sh
index e2e6e3e..220c73e 100755
--- a/test/wdog/autorun-wdog.sh
+++ b/test/wdog/autorun-wdog.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 source /unit_tests/test-utils.sh
 
-- 
1.8.0

